Cargo.lock

```diff
@@ -557,8 +368,24 @@ version = "0.5.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7f6fd5ddaf0351dff5b8da21b2fb4ff8e08ddd02857f0bf69c47639106c0fff0"
 dependencies = [
- "asn1-rs-derive",
- "asn1-rs-impl",
+ "asn1-rs-derive 0.4.0",
```

Comment:
Where theses duplicates come from?
Can you try to avoid them by using the same versions of the dependencies that use them?

Reply:
Essentially it comes from libp2p and litep2p using different versions of x509-parser. Unfortunately I don't think we can fix it, as those are defined in polkadot-sdk.

And polkadot-sdk has the same issue in their Cargo.lock: https://github.com/paritytech/polkadot-sdk/blob/035211d707d0a74a2a768fd658160721f09d5b44/Cargo.lock#L8121

Reply:
Ho I see, is there already a GH issue on polkadot-sdk about that?

Reply:
I don't see it